app/bt/exposureNotifications.spec.ts

```diff
@@ -0,0 +1,98 @@
+import dayjs from 'dayjs';
+
+import { DateTimeUtils } from '../helpers';
+import { Possible, ExposureDatum } from '../exposureHistory';
+import { toExposureHistory, RawExposure } from './exposureNotifications';
+
+describe('toExposureHistory', () => {
+  describe('when there are no exposure notifications', () => {
+    it('returns a history of NoKnown Exposures for the past 21 days', () => {
+      const rawExposures: RawExposure[] = [];
+      const today = Date.now();
+      const twentyDaysAgo = dayjs(today).subtract(20, 'day').valueOf();
+
+      const result = toExposureHistory(rawExposures);
+
+      const lastDay = result[result.length - 1].date;
+      const firstDay = result[0].date;
+      expect(result.length).toBe(21);
+      expect(
+        result.every((datum: ExposureDatum) => datum.kind === 'NoKnown'),
+      ).toBe(true);
+      expect(dayjs(lastDay).isSame(today, 'day')).toBe(true);
+      expect(dayjs(firstDay).isSame(twentyDaysAgo, 'day')).toBe(true);
+    });
+  });
+
+  describe('when there was a possible exposure two days ago', () => {
+    it('returns a history with a PossibleExposure 2 days ago', () => {
+      const today = Date.now();
+      const twoDaysAgo = dayjs(today).subtract(2, 'day').valueOf();
+      const duration = 30 * 60 * 1000;
+      const rawExposures: RawExposure[] = [
+        {
+          id: 'ABCD-EFGH',
+          date: twoDaysAgo,
+          duration,
+          totalRiskScore: 2,
+          transmissionRiskLevel: 3,
+        },
+      ];
+      const expected: Possible = {
+        kind: 'Possible',
+        date: DateTimeUtils.beginningOfDay(twoDaysAgo),
+        duration: duration,
+        totalRiskScore: 2,
+        transmissionRiskLevel: 3,
+      };
+
+      const result = toExposureHistory(rawExposures);
+
+      expect(result[result.length - 3]).toEqual(expected);
+    });
+  });
+
+  describe('when there are multiple raw exposures on the same day', () => {
+    it('combines the raw exposure into a single possible exposure with the sum of duration and max of risk', () => {
```

Comment:
The current strategy is to just add all the total durations, a future iteration will/should do something more sophisticated at this point to determine the total risk level for the day.